### PR TITLE
Sheri's first PR

### PR DESCRIPTION
**Pull request by ssleeyrc**

### CODE REVIEW

File: Client/Component/RelatedItems/OutfitList.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
```
Comment (logan-qiu) [inline Client/Component/RelatedItems/OutfitList.jsx:2]: Ok fine it's look neat enough for me

File: webpack.config.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ const config = {
       },
       {
         test: /\.css$/i,
-        use: [stylesHandler, "css-loader", "postcss-loader"],
+        use: [stylesHandler, "style-loader", "css-loader", "postcss-loader"],
```
Comment (camjuresich) [inline webpack.config.js:34]: I don't think this is necessary.

File: jest.config.js [hunk 3]
```diff
@@ -0,0 +1,194 @@
+/*
```
Comment (camjuresich) [inline jest.config.js:1]: Please ignore this file and make another pull request.

**Review by camjuresich — CHANGES_REQUESTED** [review]
see my comments yo!
